### PR TITLE
Add default place and API to create Queues

### PR DESCRIPTION
**Pull request by ohaz**

### CODE REVIEW

File: backend/digitales_warten.py [hunk 1]
```diff
@@ -1,20 +1,15 @@
 #!/usr/bin/env python3
-
+from flask import request, abort, jsonify
 from app import app, db
 from models.slot import Slot
+from models.place import Place
```
Comment (Gronner) [inline backend/digitales_warten.py]: Resort alphabetically

File: backend/digitales_warten.py [hunk 2]
```diff
@@ -23,6 +18,24 @@ def create_slot(queue_id):
     db.session.commit()
     return str(slot.id)
 
+@app.route('/place/<place_id>/queues', methods=['POST'])
+def create_queue(place_id):
+    if 'application/json' not in request.headers['Content-Type']:
```
Comment (Gronner) [inline backend/digitales_warten.py]: Maybe access the header object using the associated API? https://werkzeug.palletsprojects.com/en/1.0.x/datastructures/#werkzeug.datastructures.EnvironHeaders

File: backend/digitales_warten.py [hunk 3]
```diff
@@ -23,6 +18,24 @@ def create_slot(queue_id):
...
+        abort(400)
+    data = request.json
+    if 'queueName' not in data:
+        abort(400)
+    place = Place.query.filter_by(id=place_id).first()
+    if place is None:
+        abort(404)
+    # Check password here
+    queueName = data['queueName']
```
Comment (Gronner) [inline backend/digitales_warten.py]: snake_case instead of CamelCase pls

File: backend/models/place.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    password = db.Column(db.String)
+
+    def __repr__(self):
+        return f'<Place {self.id}>'
+
+def generate_default_place():
+    password = 'Admin'
+    password_hash = generate_password_hash(password, 'sha256')
+    place_id = generate_place_id()
+    print(f'==== Generated default place is: {place_id}')
```
Comment (Gronner) [inline backend/models/place.py]: Mark as #DEBUG

File: backend/digitales_warten.py (hunk 2 above)
Comment (ohaz) [reply backend/digitales_warten.py]: No.

File: backend/digitales_warten.py (hunk 1 above)
Comment (ohaz) [reply backend/digitales_warten.py]: Done

File: backend/digitales_warten.py (hunk 3 above)
Comment (ohaz) [reply backend/digitales_warten.py]: Done

File: backend/models/place.py (hunk 4 above)
Comment (ohaz) [reply backend/models/place.py]: Added a debug comment to the function itself. Is this fine?

**Comment by ohaz** [comment]
Closes #26 #18

**Pull request merged**